arch/riscv/Kconfig.isa

```diff
@@ -0,0 +1,95 @@
+# Copyright (c) 2022 Carlo Caione <ccaione@baylibre.com>
+# SPDX-License-Identifier: Apache-2.0
+
+config RISCV_ISA_RV32I
+	bool
+	help
+	  RV32I Base Integer Instruction Set - 32bit
+
+config RISCV_ISA_RV32E
+	bool
+	help
+	  RV32E Base Integer Instruction Set (Embedded) - 32bit
+
+config RISCV_ISA_RV64I
+	bool
+	default y if 64BIT
+	help
+	  RV64I Base Integer Instruction Set - 64bit
+
+config RISCV_ISA_RV128I
+	bool
+	help
+	  RV128I Base Integer Instruction Set - 128bit
+
+config RISCV_ISA_EXT_M
+	bool
+	default y
```

Comment:
https://github.com/zephyrproject-rtos/zephyr/pull/45582#discussion_r871402417

Reply:
right, that slipped through